calico_containers/pycalico/util.py

```diff
@@ -90,4 +90,24 @@ def get_hostname():
     except KeyError:
         # The user does not have a set $HOSTNAME. Since this is a common
         # scenario, return socekt.gethostname instead of just erroring.
-        return socket.gethostname()
\ No newline at end of file
+        return socket.gethostname()
+
+def check_ports(port_list):
+    """
+    Checks whether a list of ports are within range of 0 and 65535.
+    The port list may contains ints or strings including a a range of ports,
+    delimited by a colon.
+
+    :param port_list:
+    :return: a Boolean: True if in range, False if not in range
+    """
+    in_range = True
+    for port in port_list:
+        if ":" in str(port):
```

Comment:
Shouldn't 1:2:3 be invalid?  I think this will mark is as valid though.

Also shouldn;'t the second be higher than the first?